src/features/Account/RegisterForm.tsx

```diff
@@ -106,26 +106,57 @@ const Register = () => {
             }}
             {...formik.getFieldProps('cpassword')}
           />
-          {formik.errors.cpassword && formik.touched.cpassword ? (
-            <Typography variant="caption" color="error">
-              {formik.errors.cpassword as string}
-            </Typography>
-          ) : null}
         </Grid>
+
+        {/* Select user type (Participant or Organizer) */}
         <Grid item xs={12}>
-          <Button type="submit" variant="contained" color="primary" fullWidth 
-            sx={{
-              ":hover": {backgroundColor: '#3C343B'},
-              background: '#CF7F49',
-              color: 'white',
+          <TextField variant="outlined" fullWidth
+            error = {formik.errors.userType && formik.touched.userType ? true : false}
+            label="How will you use Rodeo Co?"
+            name="userType"
+            select
+            SelectProps={{
+              native: true,
             }}
+            {...formik.getFieldProps('userType')}
+          >
+            <option value=""></option>
+            <option value="participant">I want to enter rodeo events</option>
+            <option value="admin">I will be organizing/hosting rodeos</option>
```

Comment:
Can we change this to "I am a rodeo producer"? Sorry, I'm so picky with copy and I always go back and forth 😅